bitcoin/src/blockdata/script/instruction.rs

```diff
@@ -126,16 +128,12 @@ impl<'a> Instructions<'a> {
 
     pub(super) fn next_push_data_len(
         &mut self,
-        len: PushDataLenLen,
+        len: UintSize,
```

Comment:
I personally find the previous name more readable. But I'm not going to block the PR for it.

Reply:
I also prefer the old name. I think we should retain it. (And maybe we should even rename the method from _uint to _pushdata or something.)

Reply:
I've used the original name and renamed the function to `read_push_data_len`, re-written the commit log (and PR description), and updated the docs too.